### DISCUSSION TITLE
Copy HN Post Title along with Subject body/description

### DISCUSSION THREAD

**Discussion by trySometngNew** [SimonHalvdansson/Harmonic-HN#107] [Ideas]
Hi @SimonHalvdansson,
I am using Harmonic Android App v1.10 currently.  I am unable to copy main hn post body along with post title.
Feature suggestion : 
1. Can you provide small copy icon in pop-up modal window when user long presses on hn post text? Because it may not be intuitive for user to select text and copy it in modal window. For first few tries, I wasn't even sure what long pressing hn post achieved for me with just Done button on bottom. Or you can straightaway copy text to clipboard if user long presses hn post with success msg once copied.
2. Can you copy HN Post Title, url along with hn post Description in single text to clipboard so as to make it easy for user to share comprehensive copy of hn post?
3. In point 2 above, does it make sense to add hn post link as well to above copied clipboard text? This should make it very easy for user to add source to copied hn post text. Should save an additional step for adding hn post link.

![Screenshot_20231004-020603_Harmonic](https://github.com/SimonHalvdansson/Harmonic-HN/assets/8050894/fe483f02-589c-4110-a19e-30db608ff503)

Regards,
Sourabh

**Comment by SimonHalvdansson** [comment]
This was actually fixed a little while back and will ship in the next version, as for copying url or title that can be done with the shared buttons.
